spec/index.html

```diff
@@ -240,7 +240,27 @@ <h2>Uses of Dataset Canonicalization</h2>
       RDF store or serialization is arbitrary,
       and typically not relatable to the context within which it is used.</p>
 
-    <p>This specification defines an algorithm for creating stable <a>blank node identifiers</a> repeatably for different serializations possibly using individualized <a>blank node identifiers</a> of the same RDF graph (dataset) by grounding each <a>blank node</a> through the nodes to which it is connected, essentially creating <em>Skolem <a>blank node identifiers</a></em>. As a result, a graph signature can be obtained by hashing a canonical serialization of the resulting <a>normalized dataset</a>, allowing for the isomorphism and digital signing use cases. As blank node identifiers can be stable even with other changes to a graph (dataset), in some cases it is possible to compute the difference between two graphs (datasets), for example if changes are made only to ground triples, or if new blank nodes are introduced which do not create an automorphic confusion with other existing blank nodes. If any information which would change the generated blank node identifier, a resulting diff might indicate a greater set of changes than actually exists.</p>
+    <p>This specification defines an algorithm for creating stable
+      <a>blank node identifiers</a> repeatably for different serializations
+      possibly using individualized <a>blank node identifiers</a>
+      of the same RDF graph (dataset) by grounding each <a>blank node</a>
+      through the nodes to which it is connected,
+      essentially creating <em>Skolem <a>blank node identifiers</a></em>.
```

Comment:
```suggestion
      through the nodes to which it is connected.
```

I am not sure it is worth referring to Skolem identifiers. It is correct that the term "Skolem" exists in mathematical logic in general but, in the RDF world, we only have the term [Skolem IRIs](https://w3c.github.io/rdf-concepts/spec/#section-skolemization) which _MAY_ use blank node identifiers to replace them with stable IRIs. This is irrelevant for our algorithm, and this remark just confuses the reader imho.